src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -1043,6 +1048,20 @@ def print_fields(fields):
                 quoting=csv.QUOTE_NONE,
             )
 
+def extract_assay_expression(args):
+    """
+    Retrieve the selected data or generate SQL to retrieve the data from an expression assay in a dataset or cohort based on provided rules.
+    """
+
+    # Validating input combinations
+    input_validator = ExpressionInputsValidator(args)
```

Comment:
Class object was instantiated but none of the methods were called. Please see my other comment below. Ideally, here the code should call a method, such as `input_validator.validations_to_run`